dev-python/asynctest/asynctest-0.12.2.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{4,5,6,7} )
+
+inherit distutils-r1
+
+DESCRIPTION="Enhance the standard unittest with features for testing asyncio libraries"
+HOMEPAGE="https://github.com/Martiusweb/{$PN}/ https://pypi.org/project/${PN}/"
+SRC_URI="https://github.com/Martiusweb/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+DEPEND="
+	dev-python/setuptools[${PYTHON_USEDEP}]
+"
+
+src_prepare() {
+	default
+
+	sed -i "289,307{s/^/#/}" "test/test_selector.py" || die "Could not disable the failing test."
```

Comment:
No! Can't you see how horribly this will fail on version bump or even patch that causes line numbers to shift? Just prepend `_` to the method name, and tests will ignore it.

Reply:
Hehe, yes, I understand the implications of using line offsets as reference point for patching source code. To my defense I can say originally I tried to do it via `.patch` file which then got replaced with `sed` :). I did not know prepending `_` was OK. I've just amended my commit with this change.